javascript/index.js

```diff
@@ -14,52 +13,93 @@ let milUni = document.getElementById('milUni');
 let splits = document.getElementById('splits');
 
 function printTime() {
-  // ... your code goes here
+  setInterval(() => {
+    printMinutes();
+    printSeconds();
+  }, 100)
 }
 
 function printMinutes() {
-  // ... your code goes here
+  minDec.innerText = chronometer.twoDigitsNumber(chronometer.getMinutes())[0];
+  minUni.innerText = chronometer.twoDigitsNumber(chronometer.getMinutes())[1];
 }
 
 function printSeconds() {
-  // ... your code goes here
+  secDec.innerText = chronometer.twoDigitsNumber(chronometer.getSeconds())[0];
+  secUni.innerText = chronometer.twoDigitsNumber(chronometer.getSeconds())[1];
 }
 
+
 // ==> BONUS
 function printMilliseconds() {
-  // ... your code goes here
+
+  
 }
 
 function printSplit() {
   // ... your code goes here
+  splits.innerHTML += `<div>${chronometer.splitClick()}</div>`;
 }
 
 function clearSplits() {
-  // ... your code goes here
+
+  splits.innerHTML = '';
+
 }
 
 function setStopBtn() {
-  // ... your code goes here
+
+  chronometer.setStopBtn();
+
+
 }
 
 function setSplitBtn() {
-  // ... your code goes here
+  chronometer.setSplitBtn();
+  
 }
 
 function setStartBtn() {
-  // ... your code goes here
+  chronometer.startClick();
 }
 
 function setResetBtn() {
-  // ... your code goes here
+  chronometer.resetClick();
 }
 
+
 // Start/Stop Button
 btnLeft.addEventListener('click', () => {
-  // ... your code goes here
+
+
+  btnLeft.classList.toggle('stop')
+  btnRight.classList.toggle('split')
+  printTime();
+
+  if (btnLeft.innerText === 'START') {
+    btnRight.innerText = 'SPLIT'
+    btnLeft.innerText = 'STOP'
+    setStartBtn()
+    
+  } else {
+    btnLeft.innerText = 'START'
+    btnRight.innerText = 'RESET'
+    chronometer.stopClick();
+  }
+
+
+
+
 });
 
+
 // Reset/Split Button
 btnRight.addEventListener('click', () => {
   // ... your code goes here
 });
+  if (btnRight.innerText === 'RESET') {
+    setResetBtn();
+    clearSplits()
+  } else {
+    printSplit();
+  }
```

Comment:
Same concept here... once your functions are set up. You can make use of them here

```suggestion
  if (btnRight.classList.contains('reset')) {
    chronometer.resetClick();
    clearSplits();
    printTime();
  } else {
    printSplit();
  }
```